src/component/huskelapp/harTilgangTilHuskelapp.ts

```diff
@@ -0,0 +1,34 @@
+import { StringOrNothing } from '../../util/type/utility-types';
+import { ErrorMessage } from '../../api/utils';
+
+export const sjekkOpprettetEnhetLikInnloggetEnhet = (
+    huskelappEnhet: StringOrNothing,
+    arbeidslisteEnhet: StringOrNothing,
+    fargkategoriEnhet: StringOrNothing,
+    innloggetEnhet: StringOrNothing
+): boolean => {
+    if (huskelappEnhet != null) {
+        return huskelappEnhet === innloggetEnhet;
+    }
+    if (arbeidslisteEnhet != null) {
+        return arbeidslisteEnhet === innloggetEnhet;
+    }
+    if (fargkategoriEnhet != null) {
+        return fargkategoriEnhet === innloggetEnhet;
+    }
+    return true;
```

Comment:
Ønsker man faktisk returnere `true` hvis ingen av enhetene eksisterer? Da skal det holde at veileder har tilgang til brukers nåværende kontor?

Reply:
tanken var vel at dersom ingen er satt betyr det at du ikke har huskelapp/fargekategori eller arb.liste og da må du kunne få frem knappene for å kunne opprette. Men jeg kan legge til en eksplisitt sjekk på det og heller returnere false på slutten 👍